corehq/apps/app_manager/suite_xml.py

```diff
@@ -1099,7 +1099,7 @@ def _case_tile_template_string(self):
         with open(os.path.join(
                 os.path.dirname(__file__), "case_tile_templates", "tdh.txt"
         )) as f:
-            return f.read()
+            return unicode(f.read())
```

Comment:
I'm surprised this fixed anything (assuming you tested it and it made a difference). Since you're not declaring the encoding of the bytes read from the file, I would expect this to use the default encoding (ASCII). I think the right thing to do is `f.read().decode("utf-8")`, and `"utf-8"` will need to be changed if it is not the encoding of `tdh.txt`.